include/quicly.h

```diff
@@ -332,32 +332,54 @@ struct st_quicly_conn_streamgroup_state_t {
  */
 #define QUICLY_STATS_PREBUILT_FIELDS                                                                                               \
     struct {                                                                                                                       \
+        /**                                                                                                                        \
+         * Total number of packets received.                                                                                       \
+         */                                                                                                                        \
         uint64_t received;                                                                                                         \
+        /**                                                                                                                        \
+         * Total number of packets that failed decryption.                                                                         \
+         */                                                                                                                        \
         uint64_t decryption_failed;                                                                                                \
+        /**                                                                                                                        \
+         * Total number of packets sent.                                                                                           \
+         */                                                                                                                        \
         uint64_t sent;                                                                                                             \
+        /**                                                                                                                        \
+         * Total number of packets marked lost.                                                                                    \
+         */                                                                                                                        \
         uint64_t lost;                                                                                                             \
+        /**                                                                                                                        \
+         * Total number of packets for which an acknowledgement has been received.                                                 \
```

Comment:
Purely grammatical question, but do we need to say "for which acknowledgments have been received" as the noun of the phrase is "packets"?

Reply:
Ah, yes, that is correct, I've changed it.